client/utils/activity/renderers/submission.tsx

```diff
@@ -16,33 +16,18 @@ export const renderSubmissionUpdated = itemRenderer<SubmissionUpdatedActivityIte
 	message: ({ titles, item }) => {
 		const { payload } = item;
 		const { actor, pub, collection } = titles;
-		if (payload.status) {
-			if (payload.status.to === 'pending') {
-				return (
-					<>
-						{actor} submitted {pub} into {collection}
-					</>
-				);
-			}
-			if (payload.status.to === 'accepted') {
-				return (
-					<>
-						{actor} accepted {pub} for submission to {collection}
-					</>
-				);
-			}
-			if (payload.status.to === 'declined') {
-				return (
-					<>
-						{actor} denied {pub} for submission to {collection}
-					</>
-				);
-			}
+		const { from, to } = payload.status;
+		if (payload.status.to === 'pending') {
```

Comment:
I thought we changed this to `received`?

Reply:
this doesn't have the branch included from earlier